crypto/src/lib.rs

```diff
@@ -22,6 +22,7 @@
 #![doc = include_str!("../../README.md")]
 #![deny(missing_docs)]
 #![allow(clippy::single_component_path_imports)]
+#![allow(ambiguous_glob_reexports)]
```

Comment:
why are we allowing that?

Reply:
To spare some time and mostly because our prelude is never consumed externally so it's not a big deal